public/index.html

```diff
@@ -9,13 +9,14 @@
     <meta property="og:title" content="Kulttuurin kummilapset" />
     <meta property="og:description" content="Kaikki vuodesta 2020 eteenpäin syntyvät helsinkiläiset lapset kutsutaan kulttuurin kummilapsiksi. Kummilapset saavat kutsun vuosittain kahteen tapahtumaan, kunnes lapsi aloittaa koulun. Tapahtumat ovat maksuttomia." />
     <meta property="og:type" content="website" />
-    <meta property="og:image" content="%PUBLIC_URL%/images/kummilapsi-og-image.jpg" />
+    <meta property="og:image" content="https://kummilapset.hel.fi/images/kummilapsi-og-image.jpg" />
```

Comment:
What's wrong with Public URL ? It doesnt work or what?

Reply:
Public url just returns /, the build doesn't know how to create a fully qualified url